pkg/controller/disruption/disruption.go

```diff
@@ -706,6 +706,7 @@ func (dc *DisruptionController) trySync(ctx context.Context, pdb *policy.PodDisr
 	}
 	// We have unmamanged pods, instead of erroring and hotlooping in disruption controller, log and continue.
 	if len(unmanagedPods) > 0 {
+		dc.recorder.Eventf(pdb, v1.EventTypeWarning, "UnmanagedPods", "Found unmanaged pods: %v", strings.Join(unmanagedPods, ","))
```

Comment:
We were aiming for something more descriptive and also I do not think we should be referencing the pods in an event due to elevating privileges.


Probably should look more like this:

```suggestion
		dc.recorder.Eventf(pdb, v1.EventTypeWarning, "UnmanagedPods", "Pods selected by this PodDisruptionBudget were found to be unmanaged or managed by an unsupported controller (missing a /scale subresource). As a result, the status of the PDB cannot be calculated correctly, which may result in undefined behavior. To account for these pods please set \".spec.minAvailable\" field of the PDB to an integer value. Found PDB pod selector: %#v", pdb.Spec.Selector)
```

Reply:
not sure if we should include the PDB selector